frontend/src/lib-components/atoms/RoundIcon.tsx

```diff
@@ -138,12 +139,19 @@ const RoundIcon = React.memo(function RoundIcon({
   label,
   bubble,
   bubblecolor,
-  number
+  number,
+  iconColor
 }: RoundIconProps) {
+  const { colors } = useTheme()
+
   function onKeyDown(e: React.KeyboardEvent) {
     if (onClick && (e.key === ' ' || e.key === 'Enter')) onClick(e)
   }
 
+  const readableIconColor = iconColor
+    ? readableColor(iconColor, colors.grayscale.g0, colors.grayscale.g100)
```

Comment:
I'm pretty sure you're supposed to use `color` here, so it should be `const readableIconColor = readableColor(color, colors.grayscale.g0, colors.grayscale.g100)`. So `iconColor` prop can just be removed.